etc/org.opencastproject.organization-mh_default_org.cfg

```
@@ -132,6 +132,13 @@ prop.org.opencastproject.admin.mediamodule.url=${prop.org.opencastproject.engage
 #
 #prop.org.opencastproject.admin.feedback.url=
 
+# Flag to display imprint and privacy in admin-UIs footer
+#
+# Value: <boolean>
+# Default: false
+#
+prop.org.opencastproject.admin.display_about=false
```

Defaults should be commented out by default, like this

```suggestion
#prop.org.opencastproject.admin.display_about=false
```

Otherwise looks good to me.